net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Delayed delivery notification for qmail"
+SRC_URI="http://untroubled.org/qmail-notify/archive/${P}.tar.gz"
+HOMEPAGE="http://untroubled.org/qmail-notify/"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~hppa ~ppc ~sparc ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	virtual/cron
+	virtual/qmail
+"
+
+src_prepare() {
+	eapply_user
+
+	echo "$(tc-getCC) ${CFLAGS}" > conf-cc
+	echo "$(tc-getCC) ${LDFLAGS}" > conf-ld
+	sed -e "#'ar #'$(tc-getAR) #" -e "s#'ranlib #'$(tc-getRANLIB) #" -i Makefile
```

Comment:
This needs to be appended with `|| die` in case it fails to run.

Reply:
Thanks, it is fixed.